userguide/storage.rst

```diff
@@ -239,18 +240,18 @@ intended use:
   volume (pool) is created on top of the encrypted devices. Data is
   encrypted as it is written and decrypted as it is read.
 
-* The type of encryption used by %brand% is primarily used to protect 
-  data against the risks of disks (or the entire system) being physically
-  stolen, or removed to view their contents when the system is turned off.
+* The type of encryption used by %brand% is primarily intended to protect
```

Comment:
I've considerably refactored this part as a result. I've mergewd it withj thew notew at the top of the section, to removce duplication and to be clearer "up front", and also it seemed a sensible place to rewference GELI as the underlying scheme, rather than talk about encryption and only later say what is used.  Should be more straightforward.